templates/ticket/message.html

```diff
@@ -20,7 +20,9 @@
             </span>
         </div>
         <div class="content content-description">
-            {{ message.body|markdown('ticket', MATH_ENGINE)|reference|str|safe }}
+            <div class="ticket-body">
```

Comment:
Why a new div and not just put this as an additional class on the outer div?

Reply:
We use nested `div`s for comments (but that has a functional purpose); should still add as an additional class on the outer `div`?

Reply:
I don't see the point of the extra layer of div. Just put it on the parent div.

Reply:
I also don't see the point of this new div. What is the functional purpose of nested divs in comments, and is that needed here?